Plugins/Cuda/include/Acts/Plugins/Cuda/Seeding/SeedFinder.ipp

```diff
@@ -77,9 +76,9 @@ SeedFinder<external_spacepoint_t, Acts::Cuda>::createSeedsForGroup(
   // Algorithm 0. Matrix Flattening
   //---------------------------------
 
-  std::vector<Acts::InternalSpacePoint<external_spacepoint_t>*> middleSPvec;
-  std::vector<Acts::InternalSpacePoint<external_spacepoint_t>*> bottomSPvec;
-  std::vector<Acts::InternalSpacePoint<external_spacepoint_t>*> topSPvec;
+  std::vector<const external_spacepoint_t*> middleSPvec;
+  std::vector<const external_spacepoint_t*> bottomSPvec;
+  std::vector<const external_spacepoint_t*> topSPvec;
```

Comment:
we run on const inputs and all our methods are marked `const`. We need however to constantly update the quality and deltaR values since they are needer for seed filtering

Reply:
I think this means these objects just can't be passed around as const. 

I really think we should not have const objects that we mutate in a hidden way.

Reply:
The input is a `const std::vector<const external_spacepoint_t*>` since we retrieve from SG only const objects.

Reply:
Isn't that even worse? We can't modify objects from SG safely, can we?